configs/vitest.config.spec.ts

```diff
@@ -3,7 +3,7 @@ import {defineProject} from "vitest/config";
 
 export const specProjectMinimal = defineProject({
   test: {
-    name: "spec",
+    name: "spec-minimal",
```

Comment:
I decided it's better to be explicit about what preset we wanna use but this is debatable, however there is no real downside using a more specific name for these projects other than longer log prefix when executing tests